src/main/java/Server/MopeLSPServerLauncher.java

```diff
@@ -77,14 +87,18 @@ public void launchServer() {
         return ;
     }
 
+    /**
+     * <p>Stops the server by userinput </p>
```

Comment:
Which kind of user input? Please specify that the method waits for the user to hit enter, or more specifically until any kind of input is available from `System.in`.